KubeArmor/policy/policy.go

```diff
@@ -18,13 +18,19 @@ type PolicyServer struct {
 	pb.PolicyServiceServer
 	UpdateContainerPolicy func(tp.K8sKubeArmorPolicyEvent) pb.PolicyStatus
 	UpdateHostPolicy      func(tp.K8sKubeArmorHostPolicyEvent) pb.PolicyStatus
+	ContainerPolicyEnabled bool
+    HostPolicyEnabled      bool
 }
 
 // ContainerPolicy accepts container events on gRPC and update container security policies
 func (p *PolicyServer) ContainerPolicy(c context.Context, data *pb.Policy) (*pb.Response, error) {
-	policyEvent := tp.K8sKubeArmorPolicyEvent{}
 	res := new(pb.Response)
-
+	if !p.ContainerPolicyEnabled {
+        res.Status = pb.PolicyStatus_Invalid
```

Comment:
Let's add a new status to the [Policy proto](https://github.com/kubearmor/KubeArmor/blob/main/protobuf/policy.proto) for this?
Maybe something along the name of `NotEnabled`?

Reply:
I have done the required change @DelusionalOptimist